.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
+
+
+## Code Review Checklist
+ * Does the PR describe what changes are being made?
+ * Does the code follow the [Effective Go style guide](https://golang.org/doc/effective_go.html)?
+ * Does the new code have enough documentation? (comments and or diagrams)
+ * Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
```

Comment:
maybe a check something like:
```
* Would two similar additional PR-s make the code hard to maintain? (In case it modifies existing code)
```

Reply:
what do you mean?

Reply:
i added "Can this PR be broken into smaller PRs?" which loses some fidelity from what i think you mean but should lead to the same outcome? or i misunderstood completely

Reply:
It's about trying to predict whether this PR should be followed up by a refactor to make the code clearer. Or the refactor should be done before the PR.

This is about trying to figure out whether the PR is adding complexity without trying to clarify the code structure, before the making the change is difficult. Not quite sure how to nicely word it.